src/zenml/integrations/gcp/__init__.py

```diff
@@ -14,12 +14,19 @@
 """
 The GCP integration submodule provides a way to run ZenML pipelines in a cloud
 environment. Specifically, it allows the use of cloud artifact stores, metadata
-stores, and an `io` module to handle file operations on Google Cloud Storage (GCS).
+stores, and an `io` module to handle file operations on Google Cloud Storage
```

Comment:
This io module doesn't exist anymore right?

Reply:
Thanks! Changed it.